test/foundry/unit/StakingRewardsV2/StakingV2RewardCalculations.t.sol

```diff
@@ -513,10 +513,10 @@ contract StakingV2RewardCalculationTests is DefaultStakingV2Setup {
             getStakingRewardsV2(user1);
 
             // check rewards
-            rewards = rewardEscrowV2.escrowedBalanceOf(user1);
+            balance = kwenta.balanceOf(user1);
+            assertEq(balance, expectedRewards);
             rewardsUsdc = usdc.balanceOf(user1);
-            assertEq(rewards, expectedRewards);
-            assertApproxEqAbs(rewardsUsdc, expectedUsdcRewards, 10);
+            assertApproxEqAbs(rewardsUsdc, expectedUsdcRewards, 50);
```

Comment:
why do we change this from 10 to 50? does that make the assert less accurate?

i just tested this locally with 10 and it still passes